tensorboard_plugin_wit/wit_plugin_loader.py

```diff
@@ -38,6 +39,14 @@ def load(self, context):
             import tensorflow
         except ImportError:
             return
+
+        # If TB version is before 2.2.0, then do not load the WIT plugin
+        # as it is already included directly in TensorBoard.
+        version_list = tensorboard.__version__.split(".")
+        if (int(version_list[0]) < 2 or
+            (int(version_list[0]) == 2 and int(version_list[1]) < 2)):
```

Comment:
Optional, but consider using `pkg_resources.parse_version` here:

```python
import pkg_resources

_MIN_TENSORBOARD_VERSION = pkg_resources.parse_version("2.2.0")

version = pkg_resources.parse_version(tensorboard.__version__)
if version < _MIN_TENSORBOARD_VERSION:
    return None
```

This way, you don’t have to do the parsing yourself, and you don’t have
to worry about edge cases (like, `2.9a0` is a valid version number, even
if we’re not super likely to use it).

Reply:
ah great didn't know about that build-in lib. tried out packaging but that required a new dep and i didn't want to add one.

used this code and retested.